src/AppBundle/Controller/TaskController.php

```diff
@@ -97,11 +97,19 @@ public function deleteTaskAction(Task $task)
 
         if ($task->getUser() == null)
         {
-            $em = $this->getDoctrine()->getManager();
-            $em->remove($task);
-            $em->flush();
+            if ($this->getUser()->getRoles() == ['ROLE_ADMIN'] || $this->getUser()->getRoles() == ['ROLE_SUPER_ADMIN'])
```

Comment:
This wont work if later you introduced a new common ROLE shared among users.
To check if a value is in an array, please consider using : in_array()

Reply:
ok commit fix pr